src/main/java/com/aws/greengrass/device/ClientDevicesAuthService.java

```diff
@@ -215,4 +231,35 @@ private void uploadCoreDeviceCAs(List<String> certificatePemList) {
                     String.format("Failed to put core %s CA certificates to cloud", thingName), e);
         }
     }
+
+    @Override
+    public void postInject() {
+        super.postInject();
+        try {
+            authorizationHandler.registerComponent(this.getName(),
+                    new HashSet<>(Collections.singletonList(SUBSCRIBE_TO_CERTIFICATE_UPDATES)));
+        } catch (AuthorizationException e) {
+            logger.atError("initialize-cda-service-authorization-error", e)
+                    .log("Failed to initialize the client device auth service with the Authorization module.");
+        }
+        greengrassCoreIPCService.setSubscribeToCertificateUpdatesHandler(context ->
+                new SubscribeToCertificateUpdatesOperationHandler(context, this));
```

Comment:
If you construct with the authorizationHandler instead, you can move `doAuthorizationForSubscribingToCertUpdates` into the handler method itself. Thoughts?